assets/javascripts/modules/autoComplete.js

```diff
@@ -319,42 +319,34 @@ var suggestionsEvent = function () {
  * @param $elem
  * @param $targetElem
  */
-var setup = function ($elem, $targetElem, suggestionFormat) {
+var setup = function ($elem, suggestionsData, $targetElem, suggestionFormat) {
```

Comment:
This make more sense in the following order:

`$elem, $targetElem, suggestionsData, suggestionFormat`